source/managing-teams/what-engineering-managers-do.html.md

```diff
@@ -0,0 +1,89 @@
+---
+title: What Engineering Managers do
+meta_description: >
+  Engineering Managers at Nebulab are responsible for overseeing the execution of client projects
+  and looking after the growth and well-being of their team.
+---
+
+At Nebulab, we strongly believe in the value of competent, supportive management.
+
+When the company was 10 people, the Directors could look after each project personally, interacting
+with clients and coaching the team. While it was a lot of fun, that kind of structure doesn't
+really scale, so we set out to find a better alternative and created the role of the Engineering
+Manager (EM).
+
+At Nebulab, EMs report directly to the Directors, and their responsibilities are: 
+
+1. **Overseeing the execution of client projects end to end.** While we provide some guidance and
+   best practices, EMs have complete freedom in how they structure and run their projects. In
+   addition to coordinating engineering work and acting as informal project managers, you may find
+   them helping out with product management, hiring, and anything else that's needed to ensure the
+   client's success.
+2. **Looking after the well-being and growth of their team.** This might mean a lot of different
+   things depending on the project and team composition. At the very least, it involves holding
+   [regular 1:1s](/managing-teams/holding-one-on-ones/) with all team members, protecting their
+   [Investment Time](/personal-growth/investment-time/) and work-life balance, looking after their 
+   career progression through our [Growth Framework](/personal-growth/) and assisting the Directors
+   with compensation reviews.
+
+Being an Engineering Manager is a constant balance between serving the client and serving your team.
+**The best EMs find ways to align the client's interests with the team's interests,** so that they
```

Comment:
This might be de-motivating to more inexperienced EMs... maybe we can rephrase it?

Certain projects might be extremely hard to manage and could be out of the hands of the EM.